frontend/assets/scss/controls.scss

```diff
@@ -0,0 +1,66 @@
+
+@use "sass:math";
+
+.control-container {
+    display: block;
+    position: relative;
+    cursor: pointer;
```

Comment:
The entire block-width container element is clickable to toggle the input state for checkboxes and radio buttons, which isn't standard behavior, at least on Firefox.

Our checkbox:
![image](https://user-images.githubusercontent.com/16471311/132430695-dbd5f2fe-6a04-4605-9474-621f10081c09.png)

Default checkbox:
![image](https://user-images.githubusercontent.com/16471311/132430838-d9d5d308-9888-4992-92c9-c4961f8cb29f.png)

Is this intentional/desired?

Reply:
Most websites do it (GitHub, CodePen) and it's standard in many UI toolkits. I meant to only make the button + text actually toggle the button; meant to come back and make that work more succinctly but figured that could wait for when we start to implement components and see what works best

Reply:
Moved to #818